jbpm-console-ng-generic/jbpm-console-ng-data-filters-client/src/test/java/org/jbpm/console/ng/df/client/popup/filter/DataSetFilterEditorTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2015 JBoss by Red Hat.
```

Comment:
Please update to use the latest copyright version.

Copyright 2016 Red Hat, Inc. and/or its affiliates.